descwl_shear_sims/sim.py

```diff
@@ -106,6 +107,11 @@ def make_sim(
         Dimensions of psf image.  Default 51
     dither: bool, optional
         Whether to dither the images at the pixel level, default False
+    dither_size: float, optional
+        The amplitude of dithering in unit of a fraction of a pixel
+        for testing pixel interpolation.
+        All se WCS will be dithered by this amount in both +x and +y directions.
```

Comment:
I think this should say "will be given random dithers with this amplitude in both..."

Reply:
Also this actually defaults to 0.5.  I think this should also follow the same convention as other keywords and have the default in the function declaration.